ambry-frontend/src/test/java/com.github.ambry.frontend/AmbryBlobStorageServiceTest.java

```diff
@@ -1674,9 +1688,11 @@ private JSONObject createRequestHeaders(ByteRange range, GetOption getOption) th
    */
   private void verifyAccountAndContainerFromBlobId(String blobId, Account expectedAccount, Container expectedContainer,
       RestServiceErrorCode expectedRestErrorCode) throws Exception {
-    RestRequest restRequest;
+    if (blobId.startsWith("/")) {
```

Comment:
why do we have to add this now ?

Reply:
I actually realized that this was needed. different callers pass in different types of blob IDs, some with the slash, some without. This was the simplest way to deal with this.